ramls/items-in-transit.json

```diff
@@ -44,6 +44,21 @@
             "description" : "The call number from the holdings record",
             "type": "string"
           },
+          "enumeration": {
```

Comment:
I think it make sense to update API version, since you're introducing new properties.

Reply:
Updated